core/src/main/java/org/radargun/stats/Statistics.java

```diff
@@ -173,9 +175,59 @@ default Statistics with(Statistics otherStats) {
     */
    <T> T getRepresentation(String operation, Class<T> clazz, Object... args);
 
+   StreamOfRequest getStreamOfRequest();
+
    class Converter extends ReflexiveConverters.ObjectConverter {
       public Converter() {
          super(Statistics.class);
       }
    }
+
+   class UniformRateLimiter {
+      long start = Long.MIN_VALUE;
+      final long intervalNs;
+      final AtomicLong opIndex = new AtomicLong();
```

Comment:
Besides `SynchronizedStatistics` all statistics classes are intentionally single-threaded - no need to use AtomicLong.